arrow/src/array/array_binary.rs

```diff
@@ -44,8 +44,11 @@ pub struct GenericBinaryArray<OffsetSize: OffsetSizeTrait> {
 }
 
 impl<OffsetSize: OffsetSizeTrait> GenericBinaryArray<OffsetSize> {
+    /// Get the data type of the array.
+    // Declare this function as `pub const fn` after
+    // https://github.com/rust-lang/rust/issues/93706 is merged.
```

Comment:
Make consistent comment style?
```suggestion
    // Get the data type of the array.
    // Declare this function as `pub const fn` after
    // https://github.com/rust-lang/rust/issues/93706 is merged.
```

Reply:
I just want to make the first line (`Get the data type of the array.`) be the doc of this function. And it is not necessary to expose the 2nd and 3rd line to users, so I use `//`.

Reply:
Oh, I see.

Reply:
Make sense.